### PR TITLE
Cleanup gae

### PR DESCRIPTION
**Pull request by abortz**
Use more sane directory structure, serving structure, reuse minimal py-webapp template, remove unneeded dependencies. Currently running on sayc.abortz.net, but I probably missed something ;-)

@eseidel

### CODE REVIEW

File: dist/gae/Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+PORT ?= 9000
+SOCK ?= :$(PORT)
+run_dev: compile_dev
+	@uwsgi --http $(SOCK) --master --processes 4 --wsgi-file app.py
+
+run_prod:
+	@uwsgi --http-socket $(SOCK) --processes 4 --wsgi-file app.py
+
+run_dev_gae: compile_dev
```
Comment (eseidelGoogle) [inline dist/gae/Makefile]: I don't think this is even possible anymore.  All GAE support was for the kbb.

File: dist/gae/app.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+runtime: python27
```
Comment (eseidelGoogle) [inline dist/gae/app.yaml]: I think this can be deleted.

Comment (eseidelGoogle) [inline dist/gae/app.yaml]: Unless this is not for GAE but for some other stack?

File: dist/gae/public/static/bidder.appcache [hunk 3]
```diff
@@ -0,0 +1,22 @@
+CACHE MANIFEST
```
Comment (eseidelGoogle) [inline dist/gae/public/static/bidder.appcache]: I don't think this ever worked, and probably can be killed.  @abarth

File: dist/gae/public/static/fight.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```
Comment (eseidelGoogle) [inline dist/gae/public/static/fight.html]: Can be removed.

File: dist/gae/public/static/priorities.html [hunk 5]
```diff
@@ -3,7 +3,7 @@
 	<head>
```
Comment (eseidelGoogle) [inline dist/gae/public/static/priorities.html]: I'm not sure this ever really worked either.

File: dist/gae/public/static/scripts/.gitignore [hunk 6]
```diff
@@ -0,0 +1 @@
+*.js
```
Comment (eseidelGoogle) [inline dist/gae/public/static/scripts/.gitignore]: Woh, advanced nested .gitignore technology!

**Comment by eseidelGoogle** [comment]
lgtm

**Comment by eseidelGoogle** [comment]
Sigh.  Wrong github account.  But you get the idea.

**Comment by abortz** [comment]
Yeah fair enough I'll kill the actual gae stuff. Seems unlikely to work due to z3 ;-)

**Comment by eseidel** [comment]
Going to let this one go after 7 years. 🤣

**Pull request closed without merging**